### PR TITLE
CA-223652: Add delay to reduce number of syslog messages

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
Add delay in between attempts to pause the same vbd so that the syslog doesn't get hard spammed for no benefit

Signed-off-by: Mark Syms mark.syms@citrix.com

### CODE REVIEW

**Comment by MarkSymsCtx** [comment]
The formatting of this function is a bit all over the place, could reformat the function as part of this change, maybe as a different commit in the set?

File: drivers/tapdisk-control.c [hunk 1]
```diff
@@ -926,6 +926,7 @@ static int
 tapdisk_control_pause_vbd(struct tapdisk_ctl_conn *conn,
 			  tapdisk_message_t *request, tapdisk_message_t * const response)
 {
+	struct timeval now, next = { 0, 0 }, interval = { 0, 10000 };
```
Comment (stefanopanella) [inline drivers/tapdisk-control.c]: Should we do this 100ms ?

Comment (MarkSymsCtx) [inline drivers/tapdisk-control.c]: it currently seems happy at 10ms but it's easy enough to change

Comment (stefanopanella) [inline drivers/tapdisk-control.c]: ok, looks good to me than

**Comment by stefanopanella** [comment]
Reviewed-by: Stefano Panella

**Comment by stefanopanella** [comment]
Reviewed-by: Stefano Panella

**Pull request merged**